controls/V-93457.rb

```diff
@@ -46,7 +46,7 @@
   application_accounts = input('application_accounts_domain')
   application_accounts_local = input('application_accounts_local')
   excluded_accounts = input('excluded_accounts_domain')
-  excluded_accounts_local = input('excluded_accounts_local')
+  excluded_accounts_local = powershell('Get-LocalUser | where {$_.SID -cmatch "S-1-5-*"} | where {$_.SID -clike "*-50*"} | where {$_.SID -cnotlike "*-50*-*"} | select SID, Name | sort -Property SID -Descending | select -Last 2 | select Name').stdout.strip
```

Comment:
The result I get for this command is:
> "Name         \r\n----         \r\nGuest        \r\nAdministrator"

So, that string becomes the value of `excluded_accounts_local`
You could split it and then remove the first two elements, but here's another way of doing it so you get an array of the account names:
```
  excluded_accounts_local_check = json(command: 'Get-LocalUser | where {$_.SID -cmatch "S-1-5-*"} | where {$_.SID -clike "*-50*"} | where {$_.SID -cnotlike "*-50*-*"} | select SID, Name | sort -Property SID -Descending | select -Last 2 | select Name | ConvertTo-Json').params
  excluded_accounts_local = []
  excluded_accounts_local_check.each do |account|
    excluded_accounts_local << account["Name"]
  end
```

@aaronlippold please let me know if you have a better way of doing this

Reply:
Added this code (after making it concise) in commit: b323730c1f0b45762fa97f417fd9c79aa1297cb9